src/Caching/StackExchangeRedis/src/Microsoft.Extensions.Caching.StackExchangeRedis.csproj

```diff
@@ -27,9 +26,4 @@
     <Compile Include="$(SharedSourceRoot)ThrowHelpers\ObjectDisposedThrowHelper.cs" LinkBase="Shared" />
     <Compile Include="$(SharedSourceRoot)CallerArgument\CallerArgumentExpressionAttribute.cs" LinkBase="Shared" />
   </ItemGroup>
-
-  <ItemGroup>
-    <AdditionalFiles Include="PublicAPI/$(TargetFramework)/PublicAPI.Shipped.txt" />
```

Comment:
If there is still public API in this project, then these files shouldn't be removed - they should still track the API that remains here

Reply:
Oh, I see, never mind. Do all 3 TFMs have the same API surface now?

Reply:
yes; we split it previously when we added the feature; reverting to prior state